source/_integrations/knx.markdown

```diff
@@ -1404,3 +1404,75 @@ sync_state:
+[![Open your Home Assistant instance and show your service developer tools.](https://my.home-assistant.io/badges/developer_services.svg)](https://my.home-assistant.io/redirect/developer_services/)
+
+### Group address can not be read
+
+Every `*_state_address` is read on startup and sequentially if not configured differently. If you see the following errors in your logs a group address could not be read by a GroupValueRead request from Home Assistant in time.
+
+```log
+> Could not sync group address '1/2/3' (Entity name - Feature)
+> Error: KNX bus did not respond in time (2.0 secs) to GroupValueRead request for: '1/2/3'
+```
+
+#### No communication object (CO) assigned to the group address (GA) has the Read-Flag set in ETS
+
+- Enable the read flag for *one* CO assigned to the GA. Possibly the one most likely to hold the current state (e.g., for climates `brightness_state_address` the according CO of the dimming actuator).
+
+#### Response Telegrams are not passing a line coupler, router or other filter in the installation
+
+- Use a dummy device in ETS for Home Assistant. These can be found in the ETS online catalog. Assign it to the line your interface connects Home Assistant to and link its communication objects to the group addresses you are using in Home Assistant. ETS will generate filter tables that are applied to your line couplers after updating their application.
+
+#### Unresponsive system
+
+- The timeout for logging the errors (2 seconds) is started when the GroupValueRead request is scheduled to be sent. On systems experiencing high loads sending can be delayed (e.g. Raspberry Pi running lots of integrations at startup).
+Incoming response telegrams are always processed, so no information gets lost.
+
+### Duplicate entities
+
+If you find following error in your log you seem to have a duplicated entity in your configuration.
+
+```log
+Platform knx does not generate unique IDs. ID 1/2/3 already exists - ignoring paltform.name
+```
+
+`unique_id` for KNX entities is generated of required configuration values.
+
+- binary_sensor: `state_address`
+- climate: `temperature_address` `target_temperature_state_address` `target_temperature_address` `setpoint_shift_address`
+- cover: `move_long_address` `position_address`
+- fan: `address`
+- light: `address` or all `brightness_address` of `individual_colors`
+- notify: `address`
+- scene: `address` and `scene_number`
+- sensor: `state_address`
+- switch: `address`
+- weather: `address_temperature`
+
+There can not be multiple entities on the same platform sharing these exact group addresses, even if they differ in other configuration.
+
+### xknx.yaml configuration
+
+The configuration schema used in the YAML file configured in `config_file:` is deprecated since Home Assistant 2021.4. You can use the [xknx.yaml config converter](https://xknx.io/config-converter/) to convert it to a Home Assistant compatible `configuration.yaml` schema.
```

Comment:
```suggestion
The feature to specify a xknx configuration schema file in the Home Assistant configuration YAML file (via `config_file:`) is deprecated since Home Assistant 2021.4. You can use the [xknx.yaml config converter](https://xknx.io/config-converter/) to convert it to a Home Assistant compatible `configuration.yaml` schema.
```